.eas/workflows/pre-release-check.yaml

```
@@ -0,0 +1,22 @@
+      - opened
+
+jobs:
+  build:
+    type: build
+    params:
+      platform: ios
+      profile: preview
+  e2e:
+    type: maestro
+    needs: [build]
+    environment: preview 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)